adapters/appnexus/appnexus.go

```diff
@@ -329,7 +327,6 @@ func buildRequestImp(imp *openrtb2.Imp, appnexusExt *openrtb_ext.ExtImpAppnexus,
 			bannerCopy.Pos = adcom1.PositionBelowFold.Ptr()
 		}
 
-		// Fixes #307
```

Comment:
Please do not remove this comment. These kind of comments helps give context as to why this piece of code was added.